### PR TITLE
PS-2457 show shortDescription

### PR DESCRIPTION
**Pull request by alihabib-administrate**
Display short description instead of descrption.
https://administrate.atlassian.net/browse/PS-2457

### CODE REVIEW

File: src/Api/Search.php [hunk 1]
```diff
@@ -390,11 +391,20 @@ public static function formatCatalogueOutput($catalogue)
                     $imageUrl = $course['imageUrl'];
                 }
 
+                $summary = "";
+                if ($course['teaserDescription'] != "" && $course['teaserDescription'] != null) {
```
Comment (jadkhateradmin) [inline src/Api/Search.php]: We need to check if `isset` before cheking if its empty as `teaserDescription` is only valid for Courses.

**Review by jadkhateradmin — CHANGES_REQUESTED** [review]

File: src/Api/Search.php (hunk 1 above)
Comment (alihabib-administrate) [reply src/Api/Search.php]: no need it is always set as we are requesting it in graphql

Comment (jadkhateradmin) [reply src/Api/Search.php]: @alihabib-administrate its always set for courses not LPs

File: src/Api/Search.php [hunk 2]
```diff
@@ -25,6 +25,7 @@ class Search
                 'code',
                 'name',
                 'description',
+                'teaserDescription',
```
Comment (omaya-adm) [inline src/Api/Search.php:28]: I think we need to check the learning path short description as well because it is different. Not sure about that!

Comment (alihabib-administrate) [reply src/Api/Search.php:28]: we have already checked me and @jadkhateradmin , it does not have a short description

File: src/Api/Search.php (hunk 1 above)
Comment (alihabib-administrate) [reply src/Api/Search.php]: yeah right

File: src/Api/Search.php (hunk 2 above)
Comment (jadkhateradmin) [reply src/Api/Search.php:28]: @omaya-adm The description of the LPs is already a short description, according to my understanding of the TMS. the elaborate descriptions are on courses.

File: src/Api/Search.php [hunk 3]
```diff
@@ -390,11 +391,20 @@ public static function formatCatalogueOutput($catalogue)
                     $imageUrl = $course['imageUrl'];
                 }
 
+                $summary = $course['description'];
+                if(isset($course['teaserDescription'])){
```
Comment (jadkhateradmin) [inline src/Api/Search.php]: @alihabib-administrate this line can be `isset($course['teaserDescription']) && !empty($course['teaserDescription'])` to combine both if conditions

**Review by jadkhateradmin — CHANGES_REQUESTED** [review]

File: src/Api/Search.php (hunk 3 above)
Comment (alihabib-administrate) [reply src/Api/Search.php]: done

**Review by jadkhateradmin — APPROVED** [review]

**Pull request merged**